README.md

```diff
@@ -38,6 +38,9 @@ $ git clone https://github.com/EddieHubCommunity/EddieBot.git
 $ cd EddieBot
 $ npm install
 ```
+#### Github Docs
```

Comment:
I think this should be Discord not GitHub?

Reply:
yes just replaced Github with Discord .